pkg/types/openstack/platform.go

```diff
@@ -29,6 +29,12 @@ type Platform struct {
 	// Existing Floating IP to associate with the OpenStack load balancer.
 	LbFloatingIP string `json:"lbFloatingIP"`
 
+	// ExternalDNS
+	// IP addresses of dns servers that will be added to the dns resolution
+	// of all instances in the cluster
```

Comment:
nit: This comment as well as the comments for the rest of the fields except `DefaultMachinePlatform` is not following the [best practice](https://golang.org/doc/effective_go.html#commentary). It should be a complete sentence--rather than having the field name on its own line--and end with a period. Following this convention allows automatic generation of documentation using `doc` command.

Reply:
I can open a seperate PR to address this, if thats ok